robotiq_description/urdf/robotiq_gripper.ros2_control.xacro

```diff
@@ -13,6 +13,10 @@
         <ros2_control name="${name}" type="system">
             <!-- Plugins -->
             <hardware>
+
+                <!-- Set use_dummy to true to connect to a dummy driver for testing purposes. -->
+                <param name="use_dummy">false</param>
```

Comment:
Should this be `use_mock` I thought there was a discussion to unify the language in another project